packages/wonder-blocks-core/docs.md

```diff
@@ -22,6 +26,12 @@ const styles = StyleSheet.create({
     <Text style={styles.text}>
         Hello, world!
     </Text>
+    <View style={[
+        styles.container,
+        {backgroundColor: "goldenrod", padding: 4},
+    ]}>
+        Even with an array and inline styles!
+    </View>
```

Comment:
Extended the example to show combining classes and inline styles, demonstrating this feature working.